salt/modules/mssql.py

```diff
@@ -232,7 +275,7 @@ def role_remove(role, **kwargs):
         return 'Could not create the role: {0}'.format(e)
 
 
-def login_exists(login, **kwargs):
+def login_exists(login, domain=None, **kwargs):
```

Comment:
We also need to modify the function documentation below to show that this should be passed a string, or perhaps even better just make `''` the default as well.